processor/src/main/java/com/linecorp/decaton/processor/runtime/ProcessingContextImpl.java

```diff
@@ -77,6 +77,11 @@ public String subscriptionId() {
         return subscriptionId;
     }
 
+    @Override
+    public Object trace() {
```

Comment:
WRT https://github.com/line/decaton/pull/69/files#diff-0643ff76b830bb4b4a3eeb61dc5e1563R54 this comment, the return type can be at least `TracingContext` so that users will know that the object is at least an implementation of `TracingContext` ?

Or if we assume all existing instrumentation libraries to rely on thread local, and all users need in DecatonProcessor is that, then we might not need to expose anything through `ProcessingContext` so far.